examples/app.py

```diff
@@ -85,7 +88,7 @@
     CELERY_EAGER_PROPAGATES_EXCEPTIONS=True,
     CELERY_RESULT_BACKEND="cache",
     DEBUG=True,
-    SEARCH_UI_SEARCH_API='invenio_search_ui.api',
+    SEARCH_UI_SEARCH_API='invenio_records_rest.recid_list',
```

Comment:
@lnielsen wouldn't be a solution to add custom handler for missing urls that tries to generate the url with other application object?